web_src/js/modules/fomantic/dimmer.js

```diff
@@ -3,8 +3,9 @@ import {queryElemChildren} from '../../utils/dom.js';
 
 export function initFomanticDimmer() {
   // stand-in for removed dimmer module
-  $.fn.dimmer = function (arg0, $el) {
+  $.fn.dimmer = function (arg0, arg1) {
     if (arg0 === 'add content') {
+      const $el = arg1;
       const existingDimmer = document.querySelector('body > .ui.dimmer');
       if (existingDimmer) {
         queryElemChildren(existingDimmer, '*', (el) => el.remove());
```

Comment:
```suggestion
        queryElemChildren(existingDimmer, '*', (el) => el.classList.add('hidden'));
```

Then fix #30815

Reply:
I checked behaviour on fomantic site, and yes, they do appear to just move modals, never remove them.

<img width="856" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/67904cf2-741f-4b91-a025-9a5498d58e6d">